testnets/saagetestnet/chain.json

```diff
@@ -1,52 +1,62 @@
 {
   "$schema": "../../chain.schema.json",
-  "chain_name": "saagetestnet",
+  "chain_name": "saagetestnet5",
   "status": "live",
   "network_type": "testnet",
-  "pretty_name": "Saage-Testnet",
-  "chain_id": "saage-internal-testnet-1.3",
-  "bech32_prefix": "saage",
+  "pretty_name": "Saage-Testnet-5",
+  "chain_id": "sit-5",
+  "bech32_prefix": "sge",
```

Comment:
could we please register any bech32 prefixes (might as well register mainnet's at the same time) to SLIP173 repo https://github.com/satoshilabs/slips/blob/master/slip-0173.md